wake/build.wake

```diff
@@ -59,3 +57,25 @@ global def e300ArtyDUTPlan =
     Nil
   makeRocketChipDUTPlan name e300ScalaModule testharness ""
   | setRocketChipDUTPlanConfigs configs
+
+def devKitFrequencyConfig = match _
+  25  = "sifive.freedom.unleashed.WithDevKit25MHz"
+  50  = "sifive.freedom.unleashed.WithDevKit50MHz"
+  100 = "sifive.freedom.unleashed.WithDevKit100MHz"
+  125 = "sifive.freedom.unleashed.WithDevKit125MHz"
+  150 = "sifive.freedom.unleashed.WithDevKit150MHz"
+  200 = "sifive.freedom.unleashed.WithDevKit200MHz"
+  frequency =
+    def _ = printLevel logError "FPGADevKit: {str frequency} is not a valid frequency"
+    "INVALID.FREQUENCY.CONFIG"
+
+global def makeFPGADevKitDUTPlan name frequency designConfigs designScalaModules =
+  def testharness = "sifive.fpgashells.shell.xilinx.VC707Shell"
```

Comment:
It probably isn't a good idea to default to this, since if someone fails to manually call `setRocketChipDUTPlanTestharness` afterwards, then it may not target the right shell.

Reply:
I've replaced this with `addFPGADevKitFrequencyConfig [frequency: Int] [plan: DUTPlan]` that adds a frequency config to an existing plan. since all this function does now add the frequency config and its corresponding ScalaModule.